web-server/v0.4/components/ListView/ControllerListView.js

```diff
@@ -21,18 +19,46 @@ class ControllerListView extends React.Component {
 
   componentDidMount() {
     this.setState({loading: true});
-    axios.get('http://es-perf44.perf.lab.eng.bos.redhat.com:9280/dsa.pbench.*/_search?search_type=count&source={ "aggs": { "run": { "terms": { "field": "controller", "size": 0 } } } }').then(res => {
-      const response = res.data.aggregations.run.buckets;
-      var controllers = [];
-      response.map(function(controller) {
-        controllers.push({key: controller.key, controller: controller.key, results: controller.doc_count});
+    let data = localStorage.getItem('controllers');
+    
+    if (!data) {
+      axios.post('http://es-perf44.perf.lab.eng.bos.redhat.com:9280/dsa.pbench.*/_search',
```

Comment:
Can we find a way to remove this hard-coded `http://es-perf44.perf.lab.eng.bos.redhat.com:9280` URL Elasticsearch prefix?